app/views/docs/rest.phtml

```diff
@@ -1,12 +1,377 @@
 <p>Appwrite supports multiple protocols for accessing the server, including <a href="/docs/rest">REST</a>, <a href="/docs/graphql">GraphQL</a>, and <a href="/docs/realtime">Realtime</a>.</p>
 
-<p>The REST API allows you to manipulate your Appwrite server through many endpoints, each representing a specific method of modification to a specific resource type.</p>
+<p>The REST API allows you to access your Appwrite server through HTTP requests, without the need for an SDK. Each endpoint in the API represents a specific operation on a specific resource.</p>
 
-<p>Requests and responses through the REST API follow a rigid structure - that is, they always expect the same structure for requests to a specific endpoint, and responses from that endpoint will also always have the same structure. </p>
+<h2><a href="/docs/rest#headers" id="headers">Headers</a></h2>
+<p>Appwrite's REST APIs expect certain headers to be included with each request:</p>
 
-<p>The specific structures used for Appwrite resources and their related endpoints can be found in the <b>References</b> section for each service.</p>
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 300px">Header</td>
+        <td style="width: 80px"></td>
+        <td>Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>X-Appwrite-Project: [PROJECT-ID]</code></td>
+        <td><span class="tag red">required</span></td>
```

Comment:
I don't see these rendered when running the homepage, you might need to tweak the `td` widths

Reply:
🤔 This shows up when I render it. I updated the screenshot above

Reply:
<img width="804" alt="Screen Shot 2022-12-20 at 12 22 32 PM" src="https://user-images.githubusercontent.com/29069505/208727850-5a8baa7f-0c7a-4b74-9073-ac5130298856.png">

Reply:
I see the table fine too 🧐 

![image](https://user-images.githubusercontent.com/1477010/208732351-02b8f89e-1d91-4100-9a05-3813c09caea6.png)

Reply:
Resolved on my end :+1: